### PR TITLE
More aggressive linebreaking

### LINKED ISSUE #1654: Result bars overflow in narrow windows
In narrow windows on the results page, the result bars can overflow the cards.
This should be changed so that the different viewport widths are correctly handled.

### PR DESCRIPTION
**Pull request by Kakadus**
Fixes #1654 

Adds more media-breakpoints for grid and raises the existing ones by one level.

### CODE REVIEW

File: evap/static/scss/components/_grid.scss [hunk 1]
```diff
@@ -77,19 +77,17 @@
...
         / auto    6rem       12rem         7rem  11rem;
 
-    @include media-breakpoint-down(md) {
+    @include media-breakpoint-down(lg) {
         grid:
-            "name semester  responsible responsible"
-            "name semester participants      result"
-            / auto    3rem         7rem       10rem;
+            "name semester responsible participants result"
+            / auto    3rem       7rem         10rem;
```
Comment (janno42) [inline evap/static/scss/components/_grid.scss:83]:
```suggestion
            / auto    3rem        7rem        6rem    9rem;
```
there seems to be a value missing. what do you think of this configuration?

File: evap/static/scss/components/_grid.scss [hunk 2]
```diff
@@ -77,19 +77,17 @@
...
-    @include media-breakpoint-down(sm) {
+    @include media-breakpoint-down(md) {
         grid:
-            "        name        name    semester"
-            " responsible responsible responsible"
-            "participants      result      result"
-            /         1fr         1fr        3rem;
+            "name semester  responsible responsible"
+            "name semester participants      result"
+            / auto    1fr         1fr       3rem;
```
Comment (janno42) [inline evap/static/scss/components/_grid.scss:90]:
```suggestion
            / auto     1fr          1fr        6rem;
```
3rem results in very small bars that are hardly useful (not your fault). what do you think about using this larger value?

**Review by janno42 — CHANGES_REQUESTED** [review]

File: evap/static/scss/components/_grid.scss (hunk 1 above)
Comment (Kakadus) [reply evap/static/scss/components/_grid.scss]: Yes, I must have forgotten to copy some lines there... Now it is the same configuration as before, except the boundaries

File: evap/static/scss/components/_grid.scss (hunk 2 above)
Comment (Kakadus) [reply evap/static/scss/components/_grid.scss]: This is true. I updated it (see last commit)

**Review by janno42 — APPROVED** [review]
✔️ Meets requirements
✔️ UI functionality checked

**Review by He3lixxx — APPROVED** [review]
Thanks for contributing!

**Pull request merged**